packages/harmony/src/components/internal/Menu.tsx

```diff
@@ -0,0 +1,48 @@
+import { ReactNode, Ref } from 'react'
+
+import { CSSObject } from '@emotion/react'
+
+import { Flex, FlexProps } from 'components/layout/Flex'
+import { Paper, PaperProps } from 'components/layout/Paper'
+import { Popup } from 'components/popup/Popup'
+import { PopupProps } from 'components/popup/types'
+
+// TODO menu label
+
+export type MenuProps = Omit<PopupProps, 'children'> & {
+  maxHeight?: CSSObject['maxHeight']
+  children: ReactNode
+  PaperProps?: Partial<PaperProps> & { css?: CSSObject }
```

Comment:
is there no `WithCSS<T>` or an existing emotion type for `{ css?: CSSObject }` we can `&`?

Reply:
great idea, added this utility, lmk what you think.